.github/workflows/release.yml

```diff
@@ -0,0 +1,63 @@
+          echo "Previous tag: ${prev_tag}"
+          echo "Current tag: ${curr_tag}"
+          echo "#### New rules:" > changes.txt
+          git log --pretty=%B ${prev_tag}..${curr_tag} | grep -E '^\s*new: ' | sort | sed -e 's%^% - %' >> changes.txt
+          echo "#### Changed rules:" >> changes.txt
+          git log --pretty=%B ${prev_tag}..${curr_tag} | grep -E '^\s*change: ' | sort | sed -e 's%^% - %' >> changes.txt
+          echo "#### Rules with fixes:" >> changes.txt
+          git log --pretty=%B ${prev_tag}..${curr_tag} | grep -E '^\s*fix: ' | sort | sed -e 's%^% - %' >> changes.txt
+          git log --pretty=%B ${prev_tag}..${curr_tag} | grep -oP 'Merge PR #\d+ from \K(@\S+)' | sort -u > authors_raw.txt
+          git log --pretty=%B ${prev_tag}..${curr_tag} | grep -oP "Co-authored-by: \K.*(?= <)" | sort -u | sed -e 's%^%@%' >> authors_raw.txt
+          LC_ALL=en_US.UTF-8 sort -u authors_raw.txt | grep -v 'dependabot\[bot\]' > authors.txt
+          echo "### Changes in this release:" > changelog.txt
+          cat changes.txt >> changelog.txt
+          echo "" >> changelog.txt
+          echo "Thanks to $(perl -pe 's%\n%, %' authors.txt | sed 's%, $%%')" >> changelog.txt
+          echo "" >> changelog.txt
+          echo "" >> changelog.txt
+          echo "#### Which Sigma rule package should I use?" >> changelog.txt
+          echo "A detailed explanation can be found in the [Releases.md](Releases.md) file. If you are new to Sigma, we recommend starting with the \"Core\" ruleset." >> changelog.txt
+          cat changelog.txt
+      - name: Build all release packages
+        run: |
+          python3 tests/sigma-package-release.py --min-status test --min-level high --rule-types generic --outfile sigma-core-${{ github.ref_name }}.zip
+          python3 tests/sigma-package-release.py --min-status test --min-level medium --rule-types generic --outfile sigma-core+-${{ github.ref_name }}.zip
+          python3 tests/sigma-package-release.py --min-status experimental --min-level medium --rule-types generic --outfile sigma-core++-${{ github.ref_name }}.zip
+          python3 tests/sigma-package-release.py --min-status experimental --min-level high --rule-types et --outfile sigma-et-addon-${{ github.ref_name }}.zip
+          python3 tests/sigma-package-release.py --min-status experimental --min-level high --rule-types generic et --outfile sigma-all-rules-${{ github.ref_name }}.zip
+      - name: Create Release with Assets
+        id: create_release
+        uses: softprops/action-gh-release@v1
+        with:
+          tag_name: ${{ github.ref }}
+          name: Release ${{ github.ref_name }}
+          body_path: changelog.txt
+          token: ${{ secrets.GITHUB_TOKEN }}
+          draft: false
+          prerelease: false
+          files: |
+            sigma-core-${{ github.ref_name }}.zip
+            sigma-core+-${{ github.ref_name }}.zip
+            sigma-core++-${{ github.ref_name }}.zip
```

Comment:
Perhaps we should use underscore to not confuse with the `-` sign from the release?

Reply:
Do you mean
`sigma_core++_${{ github.ref_name }}.zip`
or
`sigma_core++-${{ github.ref_name }}.zip`
?

Reply:
The release tag doesn't use a `-`. It is currently planned as `r<iso date>` e.g. `r2023-10-06`.

Reply:
ok now I see dashes :D

Reply:
I mean it's very subjective how it "feels" in the release page. I'll do a test with `_`s to get a direct comparison.

Reply:
dashes
![image](https://github.com/SigmaHQ/sigma/assets/79651203/b680f3fe-243d-4185-aaa9-669c5190b8a8)
vs
underscores
![image](https://github.com/SigmaHQ/sigma/assets/79651203/3cd63af1-9a4e-420e-a9ab-be2672ab24a2)

Yes, I also prefer the underscores